src/Pim/Bundle/EnrichBundle/Normalizer/ProductViolationNormalizer.php

```diff
@@ -44,12 +56,19 @@ public function normalize($violation, $format = null, array $context = [])
                 'attribute' => $productValue->getAttribute()->getCode(),
                 'locale'    => $productValue->getLocale(),
                 'scope'     => $productValue->getScope(),
-                'message'   => $violation->getMessage()
+                'message'   => $violation->getMessage(),
+            ];
+        } elseif ('identifier' === $path) {
+            $normalizedViolation = [
+                'attribute' => $this->attributeRepository->getIdentifierCode(),
```

Comment:
Yes, that's confusing and need to be addressed later.
The fact is there is still an identifier (and only one) attribute, like it is in the current version, and a corresponding product value. But this value is not saved in the "rawValues" column, but in the "identifier" column.